testsuite/tests/parsing/extended_indexoperators.ml.reference

```diff
@@ -101,7 +101,7 @@
             expression (extended_indexoperators.ml[12,226+16]..[12,226+17])
               Pexp_constant PConst_int (1,None)
         ]
-      expression (extended_indexoperators.ml[13,244+2]..[14,270+28])
+      expression (extended_indexoperators.ml[13,244+2]..[15,293+28])
```

Comment:
Is the fact that you added a new line in the test the only source of location change here, or does the change of syntactic category affect locations? Does the test pass as previously if you remove the new line you added?

Reply:
If the test source is unaltered then it passes with the same locations.